go.mod

```diff
@@ -1,4 +1,4 @@
-module go
+module github.com/Josephu/go
```

Comment:
At the moment, I had to change this to make it work for my ISOSIM build, I can change this back if you are ok with merging the code back.

Reply:
No problem, I'll take care of it. I will include 4I and some tests as well.